### PR TITLE
Remove first run page, migrate to Chromium baseapp

### PR DESCRIPTION
**Pull request by refi64**

### CODE REVIEW

File: com.google.Chrome.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ app-id: com.google.Chrome
 runtime: org.freedesktop.Platform
 runtime-version: '21.08'
 sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '21.08'
```
Comment (TheEvilSkeleton) [inline com.google.Chrome.yaml:6]:
```suggestion
base-version: 'master'
```

**Comment by TheEvilSkeleton** [comment]
bot, build

**Comment by refi64** [comment]
bot, build

**Review by TheEvilSkeleton — APPROVED** [review]
Works for me.

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
Don't forget to remove unnecessary files

**Pull request merged**